jumpscale/threesdk/threebot.py

```diff
@@ -89,10 +50,8 @@ def install(
         pers_path = f"{PERSISTENT_STORE}/{name}"
         configure = not os.path.exists(pers_path)
         if configure:
-            identity_data = ensure_identity(identity, email, words, explorer)
-            if not identity_data:
-                raise Value("Installation aborted, please enter correct identity information")
-            identity, email, words, explorer = identity_data
+            identity = IdentityManager(identity, email, words, explorer)
+            identity, email, words, explorer = identity.ask_identity()
```

Comment:
now that we have `threebot` entrypoint it would make sense to overwrite the stop/start method of container to also start the threebotserver not just the container
we could also add a method restart which would only restart the threebot server